frontend/src/services/effects.ts

```diff
@@ -11,25 +11,34 @@ import { parseSerializedServiceArea } from '../utils/formatters'
 import { boundingBox, representativePointsToGeoJSON } from '../utils/geojson'
 import { equals } from '../utils/list'
 import { getPropCaseInsensitive } from '../utils/serializers'
-import { getAdequacies, getCensusData, getRepresentativePoints, isPostGeocodeSuccessResponse, postGeocode } from './api'
+import {
+    getAdequacies, getCensusData, getRepresentativePoints,
+    getStaticAdequacies, getStaticDemographics, getStaticRPs, isPostGeocodeSuccessResponse,
+    postGeocode
+} from './api'
 import { Store } from './store'
 
+/**
+ * Determine whether or not to use the static behaviours or the dynamic ones.
+ */
+const appIsStatic = CONFIG.staticAssets.appIsStatic
+
 export function withEffects(store: Store) {
   /**
    * Update representative points when serviceAreas change.
    */
   store
     .on('serviceAreas')
     .subscribe(async serviceAreas => {
+      const selectedDataset = store.get('selectedDataset')
 
-      if (serviceAreas.length === 0) {
```

Comment:
This might affect the behavior after unselecting a dataset, no?

Reply:
True, I'm not sure why this was removed. Re-adding.

Reply:
Wait, why are we doing this again? We are just overwriting the value of `representativePoints` in the store a few lines down either way.

Reply:
Ok I understand what this did now lol. I re-added it.